api_scrape.py

```diff
@@ -1,17 +1,31 @@
-from bs4 import BeautifulSoup
-import urllib.request
-
+import requests
+from requests.exceptions import HTTPError
 
 urls = open('appIDs.txt', 'r')
 
-with urls as f:
-    url = f.readline()
-    req = urllib.request.Request(url)
-    resp = urllib.request.urlopen(req)
-    respData = resp.read()
-    print(respData)
 
+try:
+    with urls as f:
```

Comment:
Variable naming suggestion: I'm not totally sure what `f` is at first glance. Looking further ahead I see it's likely a "line"?

Reply:
Sorry, it's a common way to format and name for with/as in terms of working with files. I'll change it!

Reply:
Ah, so this is supposed to be the file. Gotcha. Yeah, I figured it was a common way. Similar things are done in Java where `f = new File()`. I tend to like more descriptive variable names, though. That's my preference but the code works as-is

Reply:
Yeah I decided to go on ahead and rename the variable to app_urls